benchmarks/benchmarks/bench_linalg.py

```diff
@@ -98,6 +98,13 @@ def time_op(self, op, typename):
         self.func(self.a)
 
 
+class LinalgNorm(Benchmark):
+    def setup(self):
+        self.array_5 = np.arange(5.)
+
+    def time_norm_small_array(self):
+        np.linalg.norm(self.array_5)
+        
```

Comment:
Could these be part of the `Linalg` class just above? Maybe even as an additional `shape` parameter to the class `params`?